templates/python-scrapy/src/spiders/title.py

```diff
@@ -32,7 +31,7 @@ def parse(self, response: Response) -> Generator[Union[TitleItem, Request], None
         # Extract and yield the TitleItem
         url = response.url
         title = response.css('title::text').extract_first()
-        yield TitleItem(url=url, title=title)
```

Comment:
And now the `TitleItem` doesn't have the `parsed_by` parameter 😄

Reply:
:see_no_evil: ... sorry